docs/tutorials/upgrading-ev3dev-from-command-line.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Upgrading Ev3dev from the Command Line
+subject: System
+---
+
+{% include icon.html type="warning" %}
+By upgrading ev3dev you are also upgrading the drivers, so there may be changes that break user software. 
```

Comment:
> saying "upgrading the kernel package" would make more sense because upgrading the kernel is specifically what upgrades the drivers.

More technically correct? Yes. More sense to us? Yes. More sense to the average "beginner" user? No. I'm shooting low, and then if someone knows more they can apply it here as well. To most people, they would view "ev3dev" as the whole installation, and associate "ev3dev version" with the generic software version, similar to a version of iOS or WIndows (or even Linux, although they wouldn't have experience with that). It's only people who actually understand what these packages are and how they interact that would make the distinction of "ev3dev version" and "ev3dev kernel version"..

In the end, if someone knows what a "kernel package" is, they probably won't need this page to help them out.